Lib/test/test_pstats.py

```diff
@@ -4,9 +4,13 @@
 from io import StringIO
 from pstats import SortKey
 from enum import StrEnum, _test_simple_enum
+import os
 
```

Comment:
```suggestion

import os
```

Let's categorize `os` in the `import`s rather than `from ... imports`

Reply:
got it